packages/instant-meilisearch/src/adapter/search-request-adapter/search-params-adapter.ts

```diff
@@ -214,19 +246,22 @@ export function adaptSearchParams(
 ): MeiliSearchMultiSearchParams {
   const meilisearchParams = MeiliParamsCreator(searchContext)
   meilisearchParams.addQuery()
-  meilisearchParams.addIndexUid()
```

Comment:
Because index uid is non-optional, I opted for stronger types, and always initiate `meiliSearchParams` with `indexUid`.